xml/ha_clvm.xml

```diff
@@ -348,6 +345,139 @@ cLVM) for more information and details to integrate here - really helpful-->
   /dev/sdb vgtest b</screen>
     </step>
    </procedure>
+   <para>To test your clustered mirror device, proceed as follows:</para>
```

Comment:
What you need to do is to add 2 more disks in the original procedures accordingly.